pkg/volume/glusterfs/glusterfs.go

```diff
@@ -221,6 +222,13 @@ func (b *glusterfsBuilder) setUpAtInternal(dir string) error {
 		options = append(options, "ro")
 	}
 
+	p := path.Join(b.glusterfs.plugin.host.GetPluginDir(glusterfsPluginName), b.glusterfs.volName)
+	if err := os.MkdirAll(p, 0750); err != nil {
+		return err
```

Comment:
I had mistakenly used `os.Mkdir` initially which was failing due to parent directories being missing. The code then silently continued without the mount option and defaulted back to the original behaviour which I was trying to track down so I changed the logic which then made my error obvious.